sdk/eventhub/event-hubs/README.md

```diff
@@ -181,6 +182,59 @@ for await (const events of consumer.getEventIterator()){
 }
 ```
 
+### Consume events from all partitions of an Event Hub
+
+To consume events for all partitions of an Event Hub, you'll create an `EventProcessor` for a specific consumer group. When an Event Hub is created, it provides a default consumer group that can be used to get started.
+
+The `EventProcessor` will delegate processing of events to a `PartitionManager` implementation that you provide, allowing your logic to focus on the logic needed to provide value while the processor holds responsibility for managing the underlying consumer operations. In our example, we will focus on building the `EventProcessor` and use a very minimal partition processor that does no actual processing.
+
+```javascript
+class SimplePartitionProcessor {
+  async processEvents(events: ReceivedEventData[]) {
+    // your code here
+  }
+
+  async processError(error: Error) {
+    // your error handler here
+  }
+
+  async initialize() {
+    // your code here
+  }
+
+  async close() {
+    // your code here
+  }
```

Comment:
For the purposes of the sample, we might want to skip the optional methods here like` initialize` and `close`